python/validator/schema.json

```diff
@@ -11,7 +11,7 @@
       "event_version": {
         "type": "string",
         "required": true,
-        "allowed": ["1.4","1.5"]
+        "allowed": ["1.4","1.5", "1.6"]
```

Comment:
This change is not needed. This file is specific to DF application. Not related to BI adapter

Reply:
Amlan, this is also used by the validator to validate the message pushed to the queue